src/utils/getNumberOfFileChanges.ts

```diff
@@ -0,0 +1,14 @@
+async function getNumberOfFileChanges(gitAPI:any, filePath:string){
+    let blames = await gitAPI?.repositories[0].blame(
+        filePath || "."
+    );
+    let commitHashArray = [] as String[];
+    let splitBlames = blames.split("\n").map((line:string) => {
+        commitHashArray.push(line.split(" ")[0]);
```

Comment:
Why do we split twice?
If I have
```
as82310 commit 1 user 1
3848asdf commit 2 user 2
as82310 commit 1 user 1
```
I can take the whole line and it will be still unique.
Why not count line breaks? (`\n`)

Reply:
Copy pasting from Slack: 

[Esteban Vargas](https://app.slack.com/team/U02779T4TDL)  [2:19 PM](https://watermelonsudo.slack.com/archives/C027L8BD2EQ/p1654283974999749)
Regarding the getNumberOfFileChanges PR:
We can either count the number of unique commit hashes (returns 9 on supabase-js's index.ts)
Or the total number of hashes (returns 31 on supabase-js's index.ts)

[1 reply](https://app.slack.com/client/T027L8BCRNG/C027L8BD2EQ/thread/C035PJWF249-1654272586.716359)
Today at 2:19 PMView thread


[Esteban Vargas](https://app.slack.com/team/U02779T4TDL)  [2:45 PM](https://watermelonsudo.slack.com/archives/C027L8BD2EQ/p1654285538862209)
Request for feedback: Whixh number makes mor sense to you?